MerlinAU.sh

```diff
@@ -1359,35 +1436,41 @@ _SendEMailNotification_()
    ! _CheckEMailConfigFileFromAMTM_ 1
    then return 1 ; fi
 
+   local CC_ADDRESS_STR=""  CC_ADDRESS_ARG=""
+
    [ -z "$FROM_NAME" ] && FROM_NAME="$ScriptFNameTag"
    [ -z "$FRIENDLY_ROUTER_NAME" ] && FRIENDLY_ROUTER_NAME="$MODEL_ID"
 
    ! _CreateEMailContent_ "$1" && return 1
 
    [ "$1" = "POST_REBOOT_FW_UPDATE_SETUP" ] && return 0
 
-   printf "\nSending email notification [$1]."
-   printf "\nPlease wait...\n"
+   if "$isInteractive"
```

Comment:
Smart, we don't need this printed everytime

Reply:
Unable to test at this time. However it's reasonable to expect this to work normally.

Reply:
Worked as expected